susemanager-frontend/susemanager-nodejs-sdk-devel/susemanager-nodejs-sdk-devel.changes

```diff
@@ -1,3 +1,5 @@
+- Add react-csv dependency
```

Comment:
Is this change correct? I don't see this anywhere else.

Reply:
no, that's a left over of a previous CSV export attempt. Nice catch!